### ISSUE TITLE
UI Issue: Double clicking on Board Activity does not expand activity dashboard 

### ISSUE THREAD

**Issue by ymodak** [breadboard-ai/breadboard#3571]
## Expected Behavior
UI Issue: I expect the Board Activity dashboard to expand to full screen on double clicking it

## Actual Behavior
The board activity widget does not expand to full screen on double clicking it

## Steps to Reproduce the Problem

1. Double click "Board Activity"
https://screenshot.googleplex.com/6yDH88zJGazWkV8

**Comment by paullewis** [comment]
Yeah, I'm undecided on this one. Because we click to drag and dock / release from dock, introducing a double click to min/max the overlay is a bit fiddly and peculiar. I'll give it some thought 🤔

**Comment by ymodak** [comment]
Sure thing. I was going through tutorials on [Build with Breadboard series](https://youtube.com/playlist?list=PLH7s1qvZKUeLampPQdBggg8LK31heBTxk&si=i7EYzwlGPZw1AjY0) and they showed activity dashboard can be expanded to full screen on double clicking it. Hence I filed an issue to report it. Thanks for taking a look.
